source/dub/commandline.d

```diff
@@ -557,12 +561,12 @@ class InitCommand : Command {
 			{
 				m_templateType = free_args[0];
 				free_args = free_args[1 .. $];
-				logInfo("Deprecated use of init type. Use --type=[vibe.d | deimos | minimal] in future.");
+				logWarn("Deprecated use of init type. Use --type=[vibe.d | deimos | minimal] in future.");
 			}
 		}
+
 		dub.createEmptyPackage(NativePath(dir), free_args, m_templateType, m_format, &depCallback);
 
-		logInfo("Package successfully created in %s", dir.length ? dir : ".");
```

Comment:
why remove this log line?

Reply:
I removed it since there's already a logInfo at the end of dub.createEmptyPackage, the previous output when running "dub init my-project" ended something like

```
Successfully initialized empty project in /home/user/things/my-project
Package successfully created in my-project
```

I found that repetitive so I thought about removing one of the two. Maybe that isn't the one to be removed, but the one in dub.createEmptyPackage?

Reply:
ah sure ok